Makefile

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make
+
+all:
+	@bundle exec jekyll serve -t --config _config.yml,_config-dev.yml &
```

Comment:
Please note that `_config-dev.yml` is only for local servers not suitable for hosted servers, I don't know how this changes this.

Reply:
Yes, it's true, but _config-dev.yml was included in _start_client.sh. Delete it?

Reply:
could a different param be made for each of the ways?

Reply:
Yes, something like `make` and `make dev`?

Reply:
yes, ok

Reply:
Ok, done.